### PR TITLE
Call unicode API on Windows

### PR DESCRIPTION
**Pull request by ken2812221**
Call Unicode API on Windows

### CODE REVIEW

**Review by donaloconnor — APPROVED** [review]
utACK https://github.com/bitcoin/bitcoin/pull/13888/commits/465f3537179ea9445eb75632994fad21a244dc35

**Comment by ken2812221** [comment]
@alexeyneu How could that happen?

**Comment by sipa** [comment]
@alexeyneu That seems to only affect whether or not command line arguments are passed in as wchar.

**Comment by ken2812221** [comment]
@alexeyneu You can see #13883 how do I solve this. Since we are using mingw. The linker will link `wmain` only if you pass `-municode` to it, but I don't do this.

**Comment by alexeyneu** [comment]
@ken2812221 yeah it does the trick in msys .  I've thinkin you pretend to solve whole issue  mentioned  
@sipa looks like it's command line related even in msvc

**Comment by laanwj** [comment]
utACK d38bf9105d33147c899117a4c20ba7872733186f

File: src/netbase.cpp [hunk 1]
```diff
@@ -17,6 +17,8 @@
 
 #ifndef WIN32
 #include <fcntl.h>
+#else
+#include <codecvt>
```
Comment (adriannavarrobetrian) [inline src/netbase.cpp]: amazing

**Pull request merged**